util/topgen/merge.py

```diff
@@ -240,6 +240,16 @@ def elaborate_instance(instance, block: IpBlock):
             raise ValueError(f'generate_dif contains invalid value {instance["generate_dif"]}')
         instance['generate_dif'] = converted_value
 
+    # An instance can either have a 'racl_mapping' or 'racl_mappings' but can't have both.
+    # 'racl_mapping' is used when the device only has a single register interface and
+    # 'racl_mappings' when there are more. Translate to always use unified racl_mappings entry.
+    racl_mapping = instance.get('racl_mapping')
+    if racl_mapping is not None:
+        if instance.get('racl_mappings') is not None:
+            raise ValueError("Cannot specify both 'racl_mapping' and 'racl_mappings'")
+        del instance['racl_mapping']
+        instance['racl_mappings'] = {None: racl_mapping}
```

Comment:
A dictionary with "None" as a key can be confusing. Is there a better name for this, like "default" or some such.

Reply:
It can but this would add an assumption on how a TLUL interface can be named / or not. I think this one was done in the same way as for the base address. I am happy to change that in a follow up PR.

Reply:
I hope you consider adding a default racl object in the top config, and when an IP doesn't assign one we give it the default. This has no name clash issues.

Reply:
We agreed on `core` being the default.